### PR TITLE
PARQUET-948: Account for API changes in ARROW-782

### PR DESCRIPTION
**Pull request by wesm**

### CODE REVIEW

**Comment by wesm** [comment]
@majetideepak Uwe is out this week, does this look OK to you?

**Comment by wesm** [comment]
This will need to be rebased after #292 goes in

File: src/parquet/arrow/writer.cc [hunk 1]
```diff
@@ -61,7 +61,7 @@ class LevelBuilder : public ::arrow::ArrayVisitor {
     array_offsets_.push_back(array.offset());                           \
     valid_bitmaps_.push_back(array.null_bitmap_data());                 \
     null_counts_.push_back(array.null_count());                         \
-    values_type_ = array.type_enum();                                   \
+    values_type_ = array.type_id();                                     \
```
Comment (majetideepak) [inline src/parquet/arrow/writer.cc]: NIT: Extra space ?

**Comment by majetideepak** [comment]
+1 LGTM

File: src/parquet/arrow/writer.cc (hunk 1 above)
Comment (wesm) [inline src/parquet/arrow/writer.cc]: This is inside a macro continuation

**Pull request closed without merging**